pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -101,6 +101,19 @@ func (m SQSMonitor) processSQSMessage(message *sqs.Message) (*EventBridgeEvent,
 	event := EventBridgeEvent{}
 	err := json.Unmarshal([]byte(*message.Body), &event)
 
+	if err == nil && len(event.DetailType) == 0 && len(event.Time) > 0 {
```

Comment:
can you conform this to the `if err != nil` style. I think it's worth the explicitness and reading left alignment as the success case instead of saving a few lines of code. 

Here's the general principle: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88